### PR TITLE
SCHEDULE FOR 4/15: drop filters content updates per post-ga design updates

### PR DESCRIPTION
**Pull request by akristen**
Updates screenshot
updates procedure
lightly modifies some content

### CODE REVIEW

**Comment by akristen** [comment]
This PR is currently in review with legal team. We cannot merge this until we get approval

**Review by homelessbirds — APPROVED** [review]

**Pull request merged**
